packages/admin-panel/__tests__/firstcase.test.js

```diff
@@ -0,0 +1,6 @@
+const { firstCase } = require('../src/useCases');
+
+// test example
+test('calling firstCase.sayHello() expected to be "done!!"', () => {
```

Comment:
@jacorreamed let's bear on mind that we should use BDD with the `given-when-then` approach.

Reply:
@gh0stl1m  fixed, and fixed